packages/react-reconciler/src/__tests__/ReactUpdaters-test.internal.js

```diff
@@ -315,7 +316,7 @@ describe('updaters', () => {
       return promise;
     });
     assertLog(['onCommitRoot']);
-    expect(allSchedulerTypes).toEqual([[null], [Suspender], [Suspender]]);
+    expect(allSchedulerTypes).toEqual([[null], [Suspender], []]);
```

Comment:
I think this is a bug. Mainly because it looks like it's the same issue as https://github.com/facebook/react/issues/22603 where unsuspending is marked as caused by "unknown".